app/controllers/internal/AccountTypeController.scala

```diff
@@ -57,6 +58,8 @@ object AccountTypeController extends AccountTypeController {
 }
 
 trait AccountTypeController extends FrontendController with Actions {
+  val extendedLoggingEnabled = Play.configuration.getBoolean("extended-logging-enabled").getOrElse(false)
```

Comment:
`Play.configuration` is deprecated as it refers to a global `Play` instance - this should be injected in. But it doesn't look like this repo has been updated to use dependency injection yet so you might not be able to do this in this piece of work without going beyond the scope of these changes.